assets/js/frontend/components/reviews/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Review from '../review';
+
+const Reviews = ( { attributes, componentId, reviews } ) => {
```

Comment:
Let's name this ReviewList

Reply:
Done in fde9e972de5bba44d64fe736d72f15705bb0de47.